package.json

```diff
@@ -49,36 +48,35 @@
     "cannon-es": "^0.9.1",
     "three-to-ammo": "^0.1.0",
     "three-to-cannon": "^4.0.0",
-    "webworkify": "^1.4.0"
+    "webpack": "^5.75.0",
+    "webpack-dev-server": "^4.11.1",
+    "webworkify-webpack": "^2.1.5"
   },
   "peerDependencies": {
     "aframe": ">=0.5.0"
   },
   "devDependencies": {
     "aframe": "~1.1.0",
-    "browserify": "^16.5.1",
-    "browserify-css": "^0.15.0",
-    "browserify-shim": "^3.8.14",
-    "budo": "^11.6.3",
     "chai": "^3.5.0",
     "chai-shallow-deep-equal": "^1.4.6",
     "envify": "^4.1.0",
     "karma": "^4.4.1",
-    "karma-browserify": "^6.1.0",
     "karma-chai-shallow-deep-equal": "0.0.4",
     "karma-chrome-launcher": "^2.2.0",
     "karma-env-preprocessor": "^0.1.1",
     "karma-firefox-launcher": "^1.3.0",
     "karma-mocha": "^1.3.0",
     "karma-mocha-reporter": "^2.2.5",
     "karma-sinon-chai": "^1.3.4",
+    "karma-webpack": "^5.0.0",
     "mocha": "^6.2.3",
     "patch-package": "^6.5.0",
     "replace": "^1.2.0",
     "sinon": "^2.4.1",
     "sinon-chai": "^2.14.0",
     "three": ">=0.125.0",
-    "uglify-es": "^3.3.9"
+    "uglify-es": "^3.3.9",
```

Comment:
You can remove the `uglify-es` dependency.
webpack is using `terser` to minify.